.github/workflows/run-dotnet-tests.yml

```diff
@@ -0,0 +1,36 @@
+name: Vault Dotnet Examples Tests
+on: [push]
+jobs:
+  Run-Tests:
+    runs-on: ubuntu-latest
+    env:
+      VAULT_ADDR: http://0.0.0.0:8200
+      VAULT_DEV_ROOT_TOKEN_ID: testtoken
+      VAULT_TOKEN: testtoken
```

Comment:
Oh I see now that you actually are using VAULT_TOKEN in one of your snippets. I used a different environment variable in the hello-world to differentiate from the fact that the Go client automatically uses VAULT_TOKEN if it's set. Is it different for VaultSharp?

Reply:
I have to double check on this, but I believe w/ VaultSharp you have to explicitly supply the token for it to work, it doesn't fall back on any value.

Reply:
Gotcha, okay well since the CI for the C# stuff is in a completely different file then I won't need to worry about that variable affecting the Go tests then. Cool!